dev/com.ibm.websphere.appserver.features/visibility/public/connectors-2.0/resources/l10n/com.ibm.websphere.appserver.connectors-2.0.properties

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+description=This feature enables the configuration of resource adapters to access Enterprise Information \
+ Systems (EIS) from applications. The configuration of a resource adapter also includes the configuration \
+ of connection factories, administered objects and activation specifications. Any resource adapter that \
+ complies with the Jakarta Connectors 2.0 specification. High performance connection pooling is also provided.
```

Comment:
Add a comma between `administered objects` and `and activation specifications` in  `The configuration of a resource adapter also includes the configuration of connection factories, administered objects and activation specifications.`

Also, add a hyphen in `High performance` (i.e., `High-performance`).